From 3c9676e0479fed5bfcadea9517d942922a6a9515 Mon Sep 17 00:00:00 2001
From: = <danworley@gmail.com>
Date: Mon, 26 May 2025 16:48:56 -0400
Subject: [PATCH 3/7] UWP: Default mods to external

---
 src/ship/Context.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/ship/Context.cpp b/src/ship/Context.cpp
index 2e25d9d0..46ed0894 100644
--- a/src/ship/Context.cpp
+++ b/src/ship/Context.cpp
@@ -208,7 +208,7 @@ bool Context::InitResourceManager(const std::vector<std::string>& archivePaths,
     }
 
     mMainPath = GetConfig()->GetString("Game.Main Archive", GetAppDirectoryPath());
-    mPatchesPath = GetConfig()->GetString("Game.Patches Archive", GetAppDirectoryPath() + "/mods");
+    mPatchesPath = GetConfig()->GetString("Game.Patches Archive", GetPathRelativeToAuxiliary("/mods"));
     if (archivePaths.empty()) {
         std::vector<std::string> paths = std::vector<std::string>();
         paths.push_back(mMainPath);
-- 
2.55.0

