From 2794b974e341de8f02ed7ee70c3d0c7162e7f3d3 Mon Sep 17 00:00:00 2001
From: = <danworley@gmail.com>
Date: Sat, 11 Oct 2025 11:27:39 -0400
Subject: [PATCH 5/7] uwp: Support prefpath fallback for aux save

---
 src/ship/Context.cpp | 11 ++++++++++-
 1 file changed, 10 insertions(+), 1 deletion(-)

diff --git a/src/ship/Context.cpp b/src/ship/Context.cpp
index 46ed0894..d86c23b7 100644
--- a/src/ship/Context.cpp
+++ b/src/ship/Context.cpp
@@ -508,7 +508,16 @@ std::string Context::GetPathRelativeToAppBundle(const std::string path) {
 }
 
 std::string Context::GetPathRelativeToAuxiliary(const std::string path) {
-    return std::string("E:/soh/") + path;
+    const std::string AUX_PATH = "E:/soh/";
+
+    if (std::filesystem::exists(AUX_PATH) && std::filesystem::is_directory(AUX_PATH)) {
+        return AUX_PATH + path;
+    } else {
+        char* prefpath = SDL_GetPrefPath("", "soh");
+        std::string fallback = std::string(prefpath) + "/" + path;
+        SDL_free(prefpath);
+        return fallback;
+    }
 }
 
 std::string Context::GetPathRelativeToAppDirectory(const std::string path, std::string appName) {
-- 
2.55.0

